neptune/new/integrations/optuna/__init__.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2021, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+try:
+    # pylint: disable=import-error
+    from neptune_optuna.impl import *
```

Comment:
I hope a * import is okay here

Reply:
Why not to import one class and two functions explicitly?
It'll be cleaner to user, who would immediately see what's available from `neptune-optuna`.

By default `*` prevents you from importing `_*` names, but you'll be still able to import `from neptune.new.integrations.optuna import Union` - not very nice :p